conductr-bundle-lib/src/main/java/com/typesafe/conductr/bundlelib/LocationService.java

```diff
@@ -51,7 +51,8 @@ public static URL getLookupUrl(String serviceName, URL fallback) throws Malforme
     }
 
     static HttpPayload createLookupPayload(String serviceLocator, String serviceName) throws MalformedURLException {
-        URL locatorUrl = new URL(serviceLocator + serviceName);
+        String serviceNameWithLeadingSlash = serviceName.startsWith("/") ? serviceName : "/" + serviceName;
```

Comment:
I think you'll need to attend to the java8 lib also...

Btw is head faster than startsWith?